bigframes/operations/aggregations.py

```diff
@@ -222,6 +226,24 @@ def output_type(self, *input_types: dtypes.ExpressionType) -> dtypes.ExpressionT
         ).output_type(input_types[0])
 
 
+@dataclasses.dataclass(frozen=True)
+class ArrayAggOp(UnaryAggregateOp):
+    name: ClassVar[str] = "arrayagg"
+
+    @property
+    def can_order_by(self):
+        return True
```

Comment:
its a more of a "must" than "can" right? At least for the purposes of non-determinism

Reply:
It's "must" for ordered mode but "can" for unordered mode. The unordered mode will introduce non-determinism.